roles/nexus_artifacts/tasks/main.yml

```diff
@@ -5,8 +5,8 @@
     status_code: [200,401]
   register: _nexus_endpoint
   until: '_nexus_endpoint.status in [200,401]'
-  retries: 30
```

Comment:
Why shorten? For new deployments, it may take a while for Nexus to come up before it's ready to configure

Reply:
Yeah this was a mistake. I didn't mean to commit this line. Thanks for catching it.